tools/hooks/scary_changes.rb

```diff
@@ -72,6 +72,20 @@ def detect_new_table_or_new_column
     EOS
   end
 
+  def detect_column_rename
+    changes = @all.grep(/^dashboard\/db\/migrate\//)
+    return if changes.empty? || !@changed_lines.include?("rename_column")
+
+    puts red <<-EOS
+
+        Looks like you are renaming a column in this migration:
```

Comment:
I don't love how I described the problem here. I'd love other suggestions on how to word this!

Reply:
Maybe something along the lines of: In production, database migrations may be applied much earlier than when application servers are upgraded.  Have you verified that the updated database schema works with the currently (previously?) deployed application version?